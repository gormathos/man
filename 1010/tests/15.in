sol_id;user_id;task_id;lang_id;score
125832;1051;2001;5;100.00
125833;1036;2004;3;0.00
125835;1093;2001;5;0.00
125838;1064;2005;1;0.00
125842;1048;2004;2;28.57
125847;1003;2003;1;95.24
125853;1020;2001;3;0.00
125860;1006;2002;1;97.96
125868;1078;2002;4;100.00
125877;1038;2005;2;15.00
125887;1027;2002;3;100.00
125898;1056;2001;3;100.00
125910;1084;2003;1;66.67
125923;1057;2002;3;71.43
125937;1017;2002;1;100.00
125952;1045;2002;5;0.00
125968;1024;2001;3;100.00
125985;1047;2002;2;0.00
126003;1086;2003;1;100.00
126022;1064;2004;3;0.00
126042;1043;2001;2;8.33
126063;1030;2005;4;65.00
126085;1052;2004;3;7.14
126108;1093;2001;1;0.00
126132;1082;2005;2;0.00
126157;1021;2005;4;0.00
126183;1038;2001;2;100.00
126210;1001;2005;3;80.00
126238;1029;2001;3;12.50
126267;1064;2002;4;16.33
126297;1088;2004;1;53.57
126328;1033;2001;2;20.83
126360;1055;2004;4;100.00
126393;1074;2001;5;0.00
126427;1060;2001;1;0.00
126462;1035;2005;3;0.00
126498;1083;2001;1;100.00
126535;1037;2002;1;100.00
126573;1085;2005;3;0.00
126612;1093;2002;5;100.00
126652;1048;2004;1;82.14
126693;1049;2003;2;100.00
126735;1003;2004;4;0.00
126778;1036;2004;2;0.00
126822;1020;2004;4;28.57
126867;1093;2001;2;50.00
126913;1048;2002;3;0.00
126960;1066;2002;2;100.00
127008;1067;2004;2;7.14
127057;1059;2003;1;4.76
127107;1027;2004;1;42.86
127158;1006;2003;1;71.43
127210;1080;2005;4;0.00
127263;1030;2005;2;50.00
127317;1032;2003;1;0.00
127372;1039;2003;5;0.00
127428;1062;2005;3;100.00
127485;1095;2001;3;100.00
127543;1041;2001;5;62.50
127602;1080;2005;5;0.00
127662;1080;2005;5;0.00
127723;1030;2002;2;16.33
127785;1069;2004;5;0.00
127848;1092;2005;2;95.00
127912;1045;2001;1;100.00
127977;1089;2003;5;100.00
128043;1088;2005;1;100.00
128110;1079;2001;5;100.00
128178;1091;2003;5;100.00
128247;1013;2002;2;69.39
128317;1055;2001;3;0.00
128388;1090;2005;1;45.00
128460;1030;2004;5;39.29
128533;1005;2002;5;100.00
128607;1064;2004;5;0.00
128682;1001;2005;4;0.00
128758;1045;2001;3;100.00
128835;1089;2003;2;100.00
128913;1071;2001;2;0.00
128992;1080;2003;5;61.90
129072;1039;2005;3;80.00
129153;1072;2003;5;23.81
129235;1041;2004;5;89.29
129318;1004;2001;4;66.67
129402;1055;2004;5;28.57
129487;1036;2005;2;0.00
129573;1100;2005;5;0.00
129660;1042;2004;5;100.00
129748;1055;2005;4;30.00
129837;1081;2001;5;0.00
129927;1032;2003;1;100.00
130018;1024;2005;5;15.00
130110;1064;2004;2;100.00
130203;1053;2004;3;32.14
130297;1095;2001;4;70.83
130392;1021;2002;1;65.31
130488;1055;2002;2;42.86
130585;1036;2003;4;19.05
130683;1028;2002;1;0.00
130782;1036;2002;3;40.82
130882;1020;2003;5;61.90
130983;1001;2003;3;33.33
131085;1091;2002;1;34.69
131188;1080;2005;2;10.00
131292;1006;2002;2;59.18
131397;1050;2003;4;14.29
131503;1070;2003;5;4.76
131610;1024;2002;4;100.00
131718;1095;2001;3;100.00
131827;1011;2004;1;0.00
131937;1056;2001;2;100.00
132048;1055;2005;4;100.00
132160;1020;2005;5;55.00
132273;1026;2005;3;70.00
132387;1036;2003;1;33.33
132502;1012;2001;3;100.00
132618;1010;2003;2;100.00
132735;1063;2002;5;69.39
132853;1082;2002;3;0.00
132972;1062;2002;3;61.22
133092;1013;2005;2;95.00
133213;1057;2001;4;0.00
133335;1020;2002;4;100.00
133458;1077;2001;1;58.33
133582;1056;2002;5;0.00
133707;1022;2002;4;0.00
133833;1094;2004;5;32.14
133960;1022;2001;3;16.67
134088;1042;2001;2;100.00
134217;1097;2001;3;83.33
134347;1050;2003;5;28.57
134478;1050;2004;4;100.00
134610;1098;2004;1;0.00
134743;1014;2005;2;85.00
134877;1031;2001;1;100.00
135012;1090;2002;4;40.82
135148;1034;2004;5;100.00
135285;1040;2003;5;0.00
135423;1028;2005;3;0.00
135562;1067;2004;1;21.43
135702;1098;2005;1;30.00
135843;1049;2005;2;0.00
135985;1005;2001;1;25.00
136128;1087;2004;3;100.00
136272;1074;2001;3;12.50
136417;1017;2005;1;0.00
136563;1012;2002;4;100.00
136710;1024;2001;2;83.33
136858;1003;2005;3;85.00
137007;1079;2002;5;100.00
137157;1089;2002;4;100.00
137308;1051;2001;2;41.67
137460;1020;2005;1;65.00
137613;1029;2002;5;0.00
137767;1098;2004;1;14.29
137922;1078;2004;1;50.00
138078;1006;2001;5;87.50
138235;1047;2001;2;41.67
138393;1050;2003;4;66.67
138552;1092;2002;3;100.00
138712;1058;2004;3;0.00
138873;1007;2001;2;0.00
139035;1076;2004;3;100.00
139198;1097;2003;2;100.00
139362;1061;2004;3;46.43
139527;1025;2003;3;100.00
139693;1088;2005;3;0.00
139860;1062;2004;5;0.00
140028;1032;2004;4;17.86
140197;1071;2001;4;0.00
140367;1060;2005;2;50.00
140538;1003;2001;4;87.50
140710;1027;2004;3;0.00
140883;1045;2003;5;0.00
141057;1032;2002;2;6.12
141232;1048;2005;4;100.00
141408;1052;2004;4;0.00
141585;1034;2004;3;14.29
141763;1026;2004;5;0.00
141942;1006;2003;2;80.95
142122;1032;2005;1;65.00
142303;1072;2003;4;100.00
142485;1037;2003;1;52.38
142668;1030;2003;4;66.67
142852;1040;2002;1;36.73
143037;1018;2003;5;90.48
143223;1080;2004;5;0.00
143410;1061;2002;3;0.00
143598;1061;2004;2;0.00
143787;1097;2005;1;55.00
143977;1059;2004;2;0.00
144168;1020;2005;2;100.00
144360;1080;2002;1;0.00
144553;1098;2003;4;61.90
144747;1052;2001;4;0.00
144942;1081;2002;4;85.71
145138;1037;2004;1;10.71
145335;1004;2001;3;0.00
145533;1042;2003;5;100.00
145732;1058;2005;3;60.00
145932;1098;2004;4;0.00
146133;1071;2004;2;0.00
146335;1033;2003;4;0.00
146538;1032;2003;3;42.86
146742;1028;2001;5;66.67
146947;1086;2001;5;79.17
147153;1055;2004;2;92.86
147360;1025;2002;4;24.49
147568;1056;2001;1;91.67
147777;1027;2005;4;60.00
147987;1016;2003;4;71.43
148198;1032;2001;2;100.00
148410;1014;2002;4;2.04
148623;1009;2004;4;42.86
148837;1045;2004;2;21.43
149052;1077;2003;5;90.48
149268;1033;2005;4;100.00
149485;1005;2003;5;100.00
149703;1007;2001;4;20.83
149922;1065;2002;4;34.69
150142;1060;2003;3;90.48
150363;1072;2005;4;15.00
150585;1085;2001;1;58.33
150808;1012;2002;4;100.00
151032;1062;2005;1;100.00
151257;1065;2002;5;0.00
151483;1017;2001;4;100.00
151710;1050;2002;3;83.67
151938;1089;2003;5;0.00
152167;1020;2004;4;92.86
152397;1070;2004;1;53.57
152628;1088;2001;4;37.50
152860;1094;2005;1;30.00
153093;1100;2002;3;100.00
153327;1039;2003;1;100.00
153562;1011;2003;5;100.00
153798;1006;2004;1;50.00
154035;1007;2004;3;100.00
154273;1028;2002;4;36.73
154512;1004;2005;1;35.00
154752;1017;2001;5;12.50
154993;1085;2003;4;9.52
155235;1087;2002;3;0.00
155478;1032;2001;5;100.00
155722;1006;2001;5;100.00
155967;1052;2003;2;0.00
156213;1047;2003;5;57.14
156460;1093;2004;3;0.00
156708;1051;2002;5;0.00
156957;1084;2004;2;0.00
157207;1096;2003;5;57.14
157458;1054;2001;5;20.83
157710;1021;2004;3;0.00
157963;1040;2005;4;50.00
158217;1020;2003;4;66.67
158472;1086;2005;2;95.00
158728;1017;2002;2;0.00
158985;1022;2002;5;53.06
159243;1030;2003;5;100.00
159502;1068;2002;1;24.49
159762;1039;2003;3;95.24
160023;1054;2005;3;0.00
160285;1026;2004;2;7.14
160548;1087;2003;5;19.05
160812;1078;2003;4;100.00
161077;1046;2004;5;100.00
161343;1042;2003;2;76.19
161610;1054;2004;3;100.00
161878;1069;2005;2;100.00
162147;1090;2002;3;100.00
162417;1005;2002;5;100.00
162688;1028;2004;2;100.00
162960;1039;2001;4;0.00
163233;1092;2004;4;0.00
163507;1027;2004;5;100.00
163782;1036;2001;1;66.67
164058;1086;2003;4;100.00
164335;1092;2002;4;2.04
164613;1086;2004;3;0.00
164892;1046;2005;5;25.00
165172;1078;2005;2;0.00
165453;1009;2003;4;100.00
165735;1033;2001;5;16.67
166018;1079;2001;3;45.83
166302;1091;2005;4;0.00
166587;1005;2003;1;95.24
166873;1020;2001;4;54.17
167160;1060;2001;3;0.00
167448;1037;2004;4;85.71
167737;1020;2004;2;78.57
168027;1013;2001;1;100.00
168318;1047;2004;4;78.57
168610;1082;2002;4;0.00
168903;1080;2002;1;100.00
169197;1039;2001;1;100.00
169492;1022;2004;1;71.43
169788;1064;2002;4;61.22
170085;1053;2004;1;50.00
170383;1098;2003;5;90.48
170682;1070;2005;4;0.00
170982;1069;2002;4;97.96
171283;1081;2001;4;91.67
171585;1100;2003;3;0.00
171888;1069;2002;5;6.12
172192;1018;2001;4;37.50
172497;1020;2005;3;85.00
172803;1100;2005;3;0.00
173110;1062;2003;5;100.00
173418;1092;2004;4;53.57
173727;1093;2005;4;70.00
174037;1017;2004;5;35.71
174348;1096;2002;3;57.14
174660;1078;2004;2;100.00
174973;1050;2001;3;16.67
175287;1067;2004;1;100.00
175602;1046;2004;3;0.00
175918;1093;2003;2;100.00
176235;1039;2004;2;100.00
176553;1081;2004;2;0.00
176872;1079;2004;3;60.71
177192;1067;2002;3;83.67
177513;1078;2002;4;0.00
177835;1044;2004;5;3.57
178158;1035;2003;5;0.00
178482;1078;2002;2;0.00
178807;1067;2002;1;0.00
179133;1057;2001;3;100.00
179460;1031;2001;3;50.00
179788;1082;2003;1;66.67
180117;1025;2001;4;45.83
180447;1074;2004;1;14.29
180778;1091;2003;2;0.00
181110;1038;2003;3;0.00
181443;1070;2005;5;95.00
181777;1007;2005;1;0.00
182112;1003;2001;2;4.17
182448;1057;2003;1;0.00
182785;1073;2005;3;5.00
183123;1082;2004;4;10.71
183462;1020;2001;3;0.00
183802;1086;2005;1;100.00
184143;1047;2003;4;0.00
184485;1002;2005;3;100.00
184828;1091;2003;5;0.00
185172;1064;2003;2;0.00
185517;1061;2003;1;100.00
185863;1027;2001;5;0.00
186210;1022;2002;5;20.41
186558;1077;2005;3;30.00
186907;1078;2003;2;100.00
187257;1067;2002;2;83.67
187608;1012;2001;4;62.50
187960;1049;2004;2;100.00
188313;1054;2002;4;87.76
188667;1081;2002;1;0.00
189022;1028;2002;5;0.00
189378;1031;2002;4;100.00
189735;1020;2004;2;82.14
190093;1096;2003;3;0.00
190452;1061;2002;1;100.00
190812;1018;2001;2;41.67
191173;1078;2005;5;0.00
191535;1020;2002;5;0.00
191898;1054;2003;4;100.00
192262;1017;2002;3;4.08
192627;1015;2002;5;81.63
192993;1004;2002;5;100.00
193360;1074;2005;5;10.00
193728;1062;2002;3;0.00
194097;1008;2005;4;45.00
194467;1001;2004;2;0.00
194838;1024;2003;1;0.00
195210;1093;2001;1;100.00
195583;1054;2001;3;70.83
195957;1048;2002;3;100.00
196332;1067;2003;2;9.52
196708;1010;2005;4;15.00
197085;1012;2005;3;100.00
197463;1019;2004;2;0.00
197842;1055;2005;2;80.00
198222;1066;2005;5;100.00
198603;1002;2001;3;83.33
198985;1078;2003;3;28.57
199368;1025;2002;1;0.00
199752;1084;2004;1;28.57
200137;1027;2001;1;100.00
200523;1022;2005;2;65.00
200910;1066;2003;5;61.90
201298;1049;2004;5;25.00
201687;1049;2004;1;0.00
202077;1093;2001;4;54.17
202468;1031;2001;2;100.00
202860;1070;2002;2;100.00
203253;1096;2003;2;95.24
203647;1095;2002;2;95.92
204042;1029;2002;5;36.73
204438;1008;2002;3;18.37
