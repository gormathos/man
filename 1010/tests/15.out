Pascal: 52 73 19 16
Python: 54 76 20 20
C++: 58 82 20 27
Java: 54 82 18 18
C#: 56 84 21 26
